### ISSUE TITLE
ghost variable?

### ISSUE THREAD

**Issue by mayork** [hoburg/turbofan#3]
@bqpd i think this is probably just me being stupid. However, when i run `engine_flight_profile_integration.py` i get the following TCS warning.

```
Warning: Constraint [C_{d_w}_Mission, CruiseSegment, CruiseP, AircraftP, WingPerformance_(0,)**6.5... >= 1.02e+10*C_{L}_Mission, CruiseSegment, CruiseP, AircraftP, WingPerformance_(0,)**16*M_Mission, Cruis...] is not tight because the left hand side evaluated to 7.58081994419e-13 dimensionless but the right hand side evaluated to 8.24402779406e-14 dimensionless (Allowable error: 0.0001%, Actual error: 8.2e+02%)
```
if i think interrogate the sol dict for the value of `C_{d_w}` I get the following
```
{C_{d_w}_Mission, ClimbSegment, ClimbP, AircraftP, WingPerformance: <Quantity([ 0.00945701  0.00946853], 'dimensionless')>, C_{d_w}_Mission, CruiseSegment, CruiseP, AircraftP, WingPerformance: <Quantity([ 0.01365655  0.0099775 ], 'dimensionless')>}
```
which doesn't match the value above. I find this very confusing because there appears to only be one instance of `Cdw`.

**Comment by mayork** [comment]
@1ozturkbe once this is resolved the engine can be integrated to the rest of the plane.

**Comment by bqpd** [comment]
I don't think I'm understanding the bug here....the difference between the Tight left hand side value and the variable value seems explained by the fact that the Tight left hand side is truncated, right? It's at least the variable to the power of 6.5, and may have other terms as well.

**Comment by mayork** [comment]
ah okay nvm i figured this out. user error. fixed via aa419fb132bb775ad8.

@1ozturkbe this is ready to integrate with the rest of the airplane now. I want to work on finishing out the engine paper and getting that sent to woody and then i'll perform the integration.
